Mayhemfile

```diff
@@ -0,0 +1,6 @@
+project: buzz
+target: bzzasm
+
+cmds:
+  - cmd: /repo/build/buzz/bzzasm /test.basm /dev/null /dev/null
```

Comment:
Can you add a quick comment on the impact of having a defect in this command?

Reply:
Sure, a defect would mean that there is a malformed assembly file that `bzzasm` does not handle gracefully. Did you want me to add a comment to the code?

Reply:
Exactly, maybe a comment above the command to understand the security impact of this finding. Is the attack vector a bzz assembly file that is being passed to the assembler?

Reply:
That's correct. Is this comment along the lines of what you wanted?

Reply:
Exactly, thank you :pray: